examples/basic_authentication/pubspec.yaml

```
@@ -0,0 +1,19 @@
+name: example
```

I think we also need to update the name. I checked the `hello_world` example and it does have an altered named, see [here](https://github.com/VeryGoodOpenSource/dart_frog/blob/d5ab3fc4ef86bf06021cfb0d443169109574f0d2/examples/hello_world/pubspec.yaml#L1).

```suggestion
name: basic_authentication
```